openstack/manila/templates/api-deployment.yaml

```diff
@@ -190,6 +190,7 @@ spec:
           resources:
             {{- toYaml .Values.pod.resources.api_statsd | trim | nindent 12 }}
           {{- end }}
+ {{- include "jaeger_agent_sidecar" . | indent 8 }}
```

Comment:
you don't want to include it in all regions. This needs to be wrapped in in/end condition.